ledger-core/test/integration/BaseFixture.hpp

```diff
@@ -94,19 +93,41 @@ class BaseFixture : public ::testing::Test {
         const std::shared_ptr<Services>& services
     );
 
-    void createWallet(
+    void createWalletInDatabase(
         const std::shared_ptr<Services>& services,
         const std::string& walletName,
         const std::string& currencyName,
         const std::shared_ptr<api::DynamicObject> &configuration
     );
 
-    void createAccount(
+    void createAccountInDatabase(
         const std::shared_ptr<Services>& services,
         const std::string &walletName,
         int32_t index
     );
 
+    template<class Account>
```

Comment:
I think we should use `typename` everywhere as it encompasses more usages, even though it’s unlikely we use anything else here. I think _one syntax to rule them all_ is better.

Reply:
I agree about __one syntax to rule them all__ is better. 

I used to write `class` in template parameter because is shorter but the main reason is `typename` keyword are a second meaning - specify to the compiler that the symbol in the class/struct is a type. 
So to resume is all about cognitive complexity and using the right keyword (even if `class` keyword has two meaning as well).